app/src/main/java/br/org/cesar/discordtime/stickysessions/data/remote/model/NoteRemote.kt

```diff
@@ -4,21 +4,21 @@ import com.google.gson.annotations.Expose
 import com.google.gson.annotations.SerializedName
 
 
-data class NoteRemote (
+data class NoteRemote(
 
-    @SerializedName("id")
-    @Expose
-    var id: String? = null,
-    @SerializedName("description")
-    @Expose
-    var description: String? = null,
-    @SerializedName("user")
-    @Expose(serialize = false)
-    var user: String? = null,
-    @SerializedName("topic")
-    @Expose
-    var topic: String? = null,
-    @SerializedName("session_id")
-    @Expose
-    var sessionId: String? = null
+        @SerializedName("id")
+        @Expose
+        @JvmField var id: String? = null,
```

Comment:
You are using @JvmField to allow using "id" directly because there are code places (in Java) that are using the properties directly instead of using getter and setter methods.

Help me to decide what is a better way: Do you think better keep the @JvmField or remove it changing all the places that use these properties to use the getter and setter methods?

Reply:
I recognized that there is java code in the project so I have added  @JvmField in order not to make any conflict, but in case the whole project will be I kotlin I recommend not to use it